packages/jupyter-ai/jupyter_ai/tests/test_extension.py

```diff
@@ -16,6 +16,9 @@
         ["--AiExtension.allowed_providers", KNOWN_LM_A],
     ],
 )
+@pytest.mark.skip(
+    reason="Reads from user's config file instead of an isolated one. Causes test flakiness during local development."
+)
```

Comment:
Could we make the configurable test server read from a null config instead of skipping? This should be possible by configuring `jp_configurable_serverapp()`:

```python
jp_configurable_serverapp(jp_nbconvert_templates, jp_environ, jp_server_config, jp_argv, jp_http_port, jp_base_url, tmp_path, jp_root_dir, jp_logging_stream, jp_asyncio_loop, io_loop)
```

As per https://pytest-jupyter.readthedocs.io/en/main/plugins/jupyter_server.html

Reply:
Unless you meant the jupyter-ai homegrown config in which case a fixture would be a good solution.

Reply:
@krassowski Yes for sure! I definitely don't want to forget about re-enabling these tests; you've given us a valuable starting point for testing the extension as a whole, and we will certainly expand this effort in the future. However, we've received some very strong demand for this feature to be included in a release as soon as it is ready. Hence I'm tracking this in a separate issue for now: https://github.com/jupyterlab/jupyter-ai/issues/447